docs/src/getting_started.md

```diff
@@ -1,41 +1,39 @@
-# Getting Started with Optimization in Julia
+# Getting Started with Optimization.jl
```

Comment:
The title of "in Julia" is a much nicer introduction to folks who are coming from outside of Julia, since this is likely to be the first page they hit. Though we may need to make an even higher level that also includes a bit of details on how to install. Catalyst.jl has done this really well.

Reply:
I think that's maybe a bit presumptive and I don't feel comfortable with that title, the tutorial is for this package so this title makes more sense than saying it shows how to do optimization in julia because then it should cover other packages that can be used to do optimization